services/lgtm/lgtm.tester.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('../service-tester');
+
+const t = new ServiceTester({ id: 'lgtm', title: 'LGTM' })
+module.exports = t;
+
+t.create('total alerts for a project')
+  .get('/alerts/g/apache/cloudstack.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'lgtm',
+    value: Joi.string().regex(/^[0-9kM.]+ alerts?$/)
+  }));
+
+t.create('missing project')
+  .get('/alerts/g/some-org/this-project-doesnt-exist.json')
+  .expectJSONTypes(Joi.object().keys({
```

Comment:
A bit of project convention: We use `.expectJSON()` if we're comparing the response to an object literal and `.expectJSONTypes()` if we're checking the type of the response keys or validating against a regex.